File: src/main/java/com/github/sorusclient/client/hud/impl/hunger/IHungerRenderer.java
package com.github.sorusclient.client.hud.impl.hunger;

public interface IHungerRenderer {
    void renderHunger(double x, double y, double scale, HeartRenderType heartRenderType);
    void renderHungerBackground(double x, double y, double scale);

    enum HeartRenderType {
        FULL,
        HALF,
        EMPTY,
    }

}

File: src/main/kotlin/com/github/sorusclient/client/hud/impl/hunger/IHungerRenderer.kt
/*
 * Copyright (c) 2022. Danterus
 * Copyright (c) 2022. Sorus Contributors
 *
 * SPDX-License-Identifier: MPL-2.0
 */

package com.github.sorusclient.client.hud.impl.hunger

interface IHungerRenderer {
    fun renderHunger(x: Double, y: Double, scale: Double, heartRenderType: HeartRenderType)
    fun renderHungerBackground(x: Double, y: Double, scale: Double)
    enum class HeartRenderType {
        FULL, HALF, EMPTY
    }
}